frontend/packages/integration-tests-cypress/tests/crud/k8-openshift-cruds.spec.ts

```diff
@@ -84,106 +88,114 @@ describe('Kubernetes resource CRUD operations', () => {
     'snapshot.storage.k8s.io~v1beta1~VolumeSnapshot',
   ]);
 
-  testObjs.forEach(({ kind, namespaced = true, humanizeKind = true }, resource) => {
-    describe(kind, () => {
-      const name = `${testName}-${_.kebabCase(kind)}`;
+  testObjs.forEach(
+    ({ kind, namespaced = true, humanizeKind = true, testI18n = false }, resource) => {
```

Comment:
Prettier changed the indents so everything below this appears to be changed, but it isn't.

Only additons are:
```
it('displays detail view for newly created resource instance', () => {
...
  if (testI18n) {
      cy.testI18n(kind, DETAILS_PAGE);
  }
...
...
it(`displays a list view for the resource`, () => {
...
  if (testI18n) {
    cy.testI18n(kind, LIST_PAGE);
  }
```